pages/thankyou.tsx

```diff
@@ -0,0 +1,10 @@
+export default function thankyou(){
```

Comment:
Same deal here.

Reply:
What is the thank you for? It would be good to be clear what success page this is for like /pages/registration/success`